include/unifex/finally.hpp

```diff
@@ -617,19 +618,19 @@ namespace unifex
       }
 
     private:
-      UNIFEX_NO_UNIQUE_ADDRESS CompletionSender completionSender_;
+      UNIFEX_NO_UNIQUE_ADDRESS std::remove_cvref_t<CompletionSender> completionSender_;
       UNIFEX_NO_UNIQUE_ADDRESS Receiver receiver_;
       bool started_ = false;
 
       // Result storage.
       union {
         // Storage for error-types that might be produced by ValueSender.
```

Comment:
Just noticed that comment should probably be updated to reference SourceSender instead of ValueSender.